daemon/main.c

```diff
@@ -75,6 +75,10 @@ main_sigint_cb(UNUSED int signum, UNUSED event_signal_t *sig, UNUSED void *data)
 	is_handling_sigint = true;
 	if (cmld_containers_stop(&exit, 0) < 0)
 		ERROR("Could not stop all containers");
+
+	if (!cmld_is_hostedmode_active()) {
+		SYNC_INFO()
+	}
```

Comment:
provide an exit() wrapper which is given to cmld_containers_stop() and put this
block into that wrapper.

Reply:
sounds good, switched to atexit